packages/web-components/src/components/link-list/link-list.scss

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '@carbon/ibmdotcom-styles/scss/themes/expressive/tokens';
+@import '@carbon/ibmdotcom-styles/scss/components/link-list/index';
+@import '../../globals/scss/vars';
+@import '../../globals/scss/box-sizing';
+
+:host(#{$dds-prefix}-link-list) {
+  .#{$prefix}--link-list {
+    padding-bottom: 0;
+  }
+
+  &:last-of-type {
+    padding-bottom: $carbon--layout-05;
+  }
+
+  .#{$prefix}--link-list__list-- {
+    &horizontal {
+      display: flex;
+
+      ::slotted(#{$dds-prefix}-link-list-item) {
+        padding-right: $carbon--layout-03;
+      }
+    }
+
+    &vertical {
+      display: flex;
+      flex-direction: column;
+
+      ::slotted(#{$dds-prefix}-link-list-item) {
+        padding-bottom: $carbon--layout-02;
+      }
+    }
+
+    &card {
+      ::slotted(#{$dds-prefix}-link-list-item) {
+        border-top: 1px solid $ui-03;
+      }
+    }
+
+    &end {
+      display: flex;
+      flex-direction: column;
+
+      ::slotted(#{$dds-prefix}-link-list-item) {
+        padding: $carbon--layout-01;
+        border-top: 1px solid $ui-03;
+        border-bottom: 1px solid $ui-03;
+        margin-top: -1px;
+        margin-left: -$carbon--layout-01;
+      }
+    }
+  }
+}
+
+:host(#{$dds-prefix}-link-list-item) {
+  @extend .#{$prefix}--link-list__list__CTA;
+
+  ::slotted(#{$dds-prefix}-card-link) {
+    @extend .#{$prefix}--card;
+
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+    margin-bottom: -1px;
+  }
+}
```

Comment:
Would you go `styles` packages and make the change there? I personally am fine with having styles defined here, but there is a push for reducing WC-specific styles.